### ISSUE TITLE
Publish Trigger emails on EVERY publish 

### ISSUE THREAD

**Issue by c-schwarze** [betheluniversity/cascade-event-publish-trigger#1]
The publish trigger was emailing all authors for events whenever we did a site publish (since feb 25, 2016).

**Comment by c-schwarze** [comment]
I just updated the publish trigger to only send emails for publishes sent from tinker.

I will test this the next time we have a chance to restart cascade (hopefully tonight)

**Comment by c-schwarze** [comment]
Testing linking a commit to a issue: #af7bc49d9fc18ce8c364e3f6dc5a831c116fed1e

**Comment by c-schwarze** [comment]
Testing linking a commit to a issue: #af7bc49

**Comment by c-schwarze** [comment]
Testing linking a commit to a issue: af7bc49

**Comment by c-schwarze** [comment]
The issue was an adjustment made in late feb. We talked with AMV. We decided to revert back to only sending emails on new submissions. AMV will talk to Brooke to see if the extra functionality is needed.

If we need the extra functionality:

the publish trigger needs to read a "Tinker Edits" hidden metadata field. If 0, do nothing. If 1, send an email and call a script. The script should be in Tinker, and simply edit the event value back to 0.
